desktop/cmp/input/SwitchInput.js

```diff
@@ -42,6 +44,23 @@ SwitchInput.propTypes = {
 //-----------------------
 // Implementation
 //-----------------------
+class Model extends HoistInputModel {
+
+    inputRef = createObservableRef();
```

Comment:
Think we need to build this inputRef into the Base class at this point.  Its going to dry up a ton of code.